clients/src/main/java/org/apache/kafka/clients/admin/AlterConfigsOptions.java

```diff
@@ -31,6 +31,27 @@
 
     private boolean validateOnly = false;
 
+    /**
+     * Set the request timeout in milliseconds for this operation or {@code null} if the default request timeout for the
+     * AdminClient should be used.
+     *
+     * This method is retained to keep binary compatibility with 0.11
+     */
+    public AlterConfigsOptions timeoutMs(Integer timeoutMs) {
+        this.timeoutMs = timeoutMs;
+        return this;
+    }
+
+    /**
+     * The request timeout in milliseconds for this operation or {@code null} if the default request timeout for the
+     * AdminClient should be used.
+     *
+     * This method is retained to keep binary compatibility with 0.11
+     */
+    public Integer timeoutMs() {
```

Comment:
I think we may not need this, just the other method, but it would be good to verify. Similar for other classes.

Reply:
@ijuma Sure. I will do manual test to verify this.

Reply:
@ijuma Thanks for the comment. You are right, this is not needed. I have updated the patch to remove this method.

I have successfully reproduced the error locally and also verified that the error can be fixed by the latest the patch in the PR. In these tests I run the following code with various kafka-clients.jar.

```
    DescribeAclsOptions options = new DescribeAclsOptions().timeoutMs(1000);
    System.out.println(options.timeoutMs());
```